PendingReleaseNotes.md

```diff
@@ -53,6 +53,7 @@ an example usage
    - If `false`, the OSDs will be assigned to a node permanently. Rook will configure Ceph's CRUSH map to support the portability.
 - Rook can now manage PodDisruptionBudgets for the following Daemons: OSD, Mon, RGW, MDS. OSD budgets are dynamically managed as documented in the [design](https://github.com/rook/rook/blob/master/design/ceph-managed-disruptionbudgets.md). This can be enabled with the `managePodBudgets` flag in the cluster CR. When this is enabled, drains on OSDs will be blocked by default and dynamically unblocked in a safe manner one failureDomain at a time. When a failure domain is draining, it will be marked as no out for a longer time than the default DOWN/OUT interval.
 - Flexvolume plugin now supports dynamic PVC expansion.
+- Rook can now manage MachineDisruptionBudgets for the OSDs. MachineDisruptionBudgets for OSDs are dynamically managed as documented in the [design](https://github.com/rook/rook/blob/master/design/ceph-openshift-fencing-mitigation.md). This can be enabled with the `manageMachineDisruptionBudgets` flag in the cluster CR.
```

Comment:
I don't think we should be referencing the design in the release notes. We should reference official Rook documentation.

Reply:
Agreed, will update this link